docker-compose.test.yml

```diff
@@ -19,10 +19,13 @@ services:
     image: mongo:4.2.0-bionic
     networks:
       - ant
+    ports:
+      - "27017:27017"
```

Comment:
This map is usefull :). In order to reduce differences between docker-compose versions, I suggest to replicate this mapping in the docker-compose.ci.yml and docker-compose.debug.yml files. Could we use environment variables? I suggest MONGO_PORT_INNER and MONGO_PORT_OUTER:

```yaml
ports:
      - "${MONGO_PORT_OUTER}:${MONGO_PORT_INNER}"
```